inc/class-envato-market-api.php

```diff
@@ -263,17 +263,38 @@ public function themes( $args = array() ) {
 		 * @return array A normalized array of values.
 		 */
 		public function normalize_theme( $theme ) {
-			return array(
-				'id' => $theme['id'],
-				'name' => ( ! empty( $theme['wordpress_theme_metadata']['theme_name'] ) ? $theme['wordpress_theme_metadata']['theme_name'] : '' ),
-				'author' => ( ! empty( $theme['wordpress_theme_metadata']['author_name'] ) ? $theme['wordpress_theme_metadata']['author_name'] : '' ),
-				'version' => ( ! empty( $theme['wordpress_theme_metadata']['version'] ) ? $theme['wordpress_theme_metadata']['version'] : '' ),
-				'description' => self::remove_non_unicode( $theme['wordpress_theme_metadata']['description'] ),
-				'url' => ( ! empty( $theme['url'] ) ? $theme['url'] : '' ),
-				'author_url' => ( ! empty( $theme['author_url'] ) ? $theme['author_url'] : '' ),
+			$normalized_theme = array(
+				'id'            => $theme['id'],
+				'name'          => ( ! empty( $theme['wordpress_theme_metadata']['theme_name'] ) ? $theme['wordpress_theme_metadata']['theme_name'] : '' ),
+				'author'        => ( ! empty( $theme['wordpress_theme_metadata']['author_name'] ) ? $theme['wordpress_theme_metadata']['author_name'] : '' ),
+				'version'       => ( ! empty( $theme['wordpress_theme_metadata']['version'] ) ? $theme['wordpress_theme_metadata']['version'] : '' ),
+				'description'   => self::remove_non_unicode( strip_tags( $theme['wordpress_theme_metadata']['description'] ) ),
+				'url'           => ( ! empty( $theme['url'] ) ? $theme['url'] : '' ),
+				'author_url'    => ( ! empty( $theme['author_url'] ) ? $theme['author_url'] : '' ),
 				'thumbnail_url' => ( ! empty( $theme['thumbnail_url'] ) ? $theme['thumbnail_url'] : '' ),
-				'rating' => ( ! empty( $theme['rating'] ) ? $theme['rating'] : '' ),
+				'rating'        => ( ! empty( $theme['rating'] ) ? $theme['rating'] : '' ),
+				'icon_url'      => '',
 			);
+
+			// No main thumbnail in API response, so we grab it from the preview array.
+			if ( empty( $normalized_theme['thumbnail_url'] ) && ! empty( $theme['previews'] ) && is_array( $theme['previews'] ) ) {
+				foreach ( $theme['previews'] as $possible_preview ) {
+					if ( ! empty( $possible_preview['landscape_url'] ) ) {
+						$normalized_theme['thumbnail_url'] = $possible_preview['landscape_url'];
```

Comment:
If the `data` object in the template is now looking for `icon_url` exclusively this will break the output.